packages/grid/enclave/attestation/server/gpu_attestation.py

```diff
@@ -1,11 +1,29 @@
+# standard imports
+# stdlib
+import io
+import re
+import sys
+
 # third party
 from loguru import logger
 from nv_attestation_sdk import attestation
 
-NRAS_URL = "https://nras.attestation.nvidia.com/v1/attest/gpu"
+# relative
+# relative imports
```

Comment:
NIT:  we could remove relative imports, as we have relative above added by the linter.